features/step_definitions/email_sign_up_steps.rb

```diff
@@ -20,3 +20,8 @@
 When /^I click on the button "(.*?)"$/ do |button_text|
   click_button button_text
 end
+
+And /^the "(.+)" option should be preselected by default$/ do |frequency|
+  # 'visible: false' because the radio buttons are 'hidden' in order to style them
+  expect(page).to have_css("input[name='frequency'][value='#{frequency}'][checked]", visible: false)
```

Comment:
Are the radio buttons definitely 'hidden' in the CSS?  I'm not seeing anything to suggest this on the production site.

Reply:
Our radio input has this CSS:

```css
.govuk-radios__input {
    cursor: pointer;
    position: absolute;
    z-index: 1;
    top: -2px;
    left: -2px;
    width: 44px;
    height: 44px;
    margin: 0;
    opacity: 0;
}
```

In this case, I think it's the `opacity: 0` that's causing `visible?` to return `false`. The [docs are very vague](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FElement:visible%3F) ("Not all drivers support CSS, so the result may be inaccurate.") but [other people seem to be able to corroborate this](https://til-engineering.nulogy.com/posts/461fe077bc-capybara-will-skip-invisible-elements-by-default).

Reply:
I see, didn't spot the `opacity` value is zero.  Seems entirely reasonable to set `visible: false` in that case.